gp-includes/router.php

```diff
@@ -28,7 +28,7 @@ public function request_uri() {
 	}
 
 	public function request_method() {
-		return gp_array_get( $_SERVER, 'REQUEST_METHOD', 'GET' );
+		return wp_unslash( gp_array_get( $_SERVER, 'REQUEST_METHOD', 'GET' ) );
```

Comment:
I'll remove it if you like, but as it doesn't add much overhead and future proofs us it seems like a reasonable thing to do.